subsystems/mvdm/ntvdm/dos/dos32krnl/process.c

```diff
@@ -803,9 +804,9 @@ WORD DosCreateProcess(IN LPCSTR ProgramName,
             CHAR ExpName[MAX_PATH];
 
             ExpandEnvironmentStringsA(AppName, ExpName, ARRAYSIZE(ExpName) - 1);
-            strcat(ExpName, "\"");         // Add double-quote before ProgramName
-            strcat(ExpName, ProgramName);  // Append Program name
-            strcat(ExpName, "\"");         // Add double-quote after ProgramName
+            StringCbCatA(ExpName, MAX_PATH, "\"");         // Add double-quote before ProgramName
```

Comment:
```suggestion
            StringCbCatA(ExpName, sizeof(ExpName), "\"");         // Add double-quote before ProgramName
```
This ensures that when the `ExpName` variable is changed, all `StringCbCatA` invocations will still be correct and don't need manual updating :)
Please note that I used `sizeof` here, because the `Cb` version of the strsafe functions is used:
* `Cb` -> Byte Count
* `Cch` -> Character count

Reply:
Done. Thanks.